src/mlpack/tests/rl_components_test.cpp

```diff
@@ -37,7 +37,7 @@ BOOST_AUTO_TEST_SUITE(RLComponentsTest)
  */
 BOOST_AUTO_TEST_CASE(SimplePendulumTest)
 {
-  const Pendulum task = Pendulum();
+  Pendulum task = Pendulum();
```

Comment:
Might be a good idea to test terminates after the maximal steps as well, we could use a super low value, to keep the runtime low.